### PR TITLE
Fluid container and spacing

### PR DESCRIPTION
**Pull request by mtribone**
Fixes container on home page and 1 column layout from CC to be container-fluid.

### CODE REVIEW

**Comment by mtribone** [comment]
screen shot 2016-10-26 at 10 25 21 am

**Comment by mtribone** [comment]
screen shot 2016-10-26 at 10 25 42 am

**Review by carolyncole — COMMENTED** [review]
Is this change just about adding "container-fluid" to the content-wrapper div?

Could we just add that via javascript?

```
document.getElementById('content-wrapper').classList.add('container-fluid');
```

Or add some TODOs at the beginning of the class with a ticket in sufia so we can stop maintaining this code at some point.

**Comment by mtribone** [comment]
We certainly could just add it via javascript. I'd have to remove the class `container` and add the class `container-fluid`

File: app/assets/javascripts/scholarsphere_layout.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+/**
```
Comment (carolyncole) [inline app/assets/javascripts/scholarsphere_layout.js:1]: @mtribone You likely need the BlackLight.onload like here: https://github.com/projecthydra/sufia/blob/master/app/assets/javascripts/sufia/batch.js#L1
and then close like here: https://github.com/projecthydra/sufia/blob/master/app/assets/javascripts/sufia/batch.js#L13

File: app/assets/javascripts/scholarsphere_layout.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+ * Add/remove a class to/from a DIV and not override a whole view
+ */
+
+document.getElementById('content-wrapper').classList.add('container-fluid');
```
Comment (carolyncole) [inline app/assets/javascripts/scholarsphere_layout.js]: You can also remove a class.  Here is the jquery syntax for adding and removing a class

`$( "#content-wrapper" ).removeClass( "container" ).addClass( "container-fluid" );`

So the entire file would look more like:

```
Blacklight.onLoad(function() {
   $( "#content-wrapper" ).removeClass( "container" ).addClass( "container-fluid" );
});
```

**Review by carolyncole — CHANGES_REQUESTED** [review]

File: app/assets/javascripts/scholarsphere_layout.js (hunk 1 above)
Comment (awead) [reply app/assets/javascripts/scholarsphere_layout.js:1]: changes approved!

**Pull request merged**